not-to-release/sources/newsgroup/groups.google.com_alt.animals_0e65f540816d780c_ENG_20041116_124800.xml.conllu

```diff
@@ -19,7 +19,7 @@
 
 # sent_id = newsgroup-groups.google.com_alt.animals_0e65f540816d780c_ENG_20041116_124800-0003
 # text = UNITED PRESS INTERNATIONAL
-1	UNITED	UNITED	PROPN	NNP	Number=Sing	2	compound	2:compound	_
+1	UNITED	UNITED	ADJ	NNP	Degree=Pos	2	amod	2:amod	_
 2	PRESS	PRESS	PROPN	NNP	Number=Sing	3	compound	3:compound	_
```

Comment:
Title case - will stop commenting on these, probably easiest to find automatically